staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_type.go

```diff
@@ -142,6 +142,7 @@ func (g *genClientForType) GenerateType(c *generator.Context, t *types.Type, w i
 		"GroupGoName":              g.groupGoName,
 		"Version":                  namer.IC(g.version),
 		"klogInfof":                c.Universe.Type(types.Name{Package: "k8s.io/klog", Name: "V"}),
+		"errorForbidden":           c.Universe.Function(types.Name{Package: "k8s.io/apimachinery/pkg/api/errors", Name: "IsForbidden"}),
```

Comment:
change errorForbidden -> errorsIsForbidden, more readable and consistent with the naming style of other keys in this map.

Reply:
Updated